app/views/like/_like.html.erb

```diff
@@ -1,92 +1,99 @@
+    <input type='hidden' name='nid' value='<%= node.id %>'/>
+    <input type='submit' class='btn btn-primary' value='Give'/> &nbsp; <a _target='blank' href='/wiki/barnstars'>Learn more</a>
+    </form>
+    <% end %>
+    
+    <% if logged_in_as(['admin', 'moderator']) || (current_user && current_user.uid == node.uid) %>
+    
+      <hr />
+    
+      <div class='btn-group'>
+        <a rel='tooltip' title='Flag as spam' class='btn btn-sm btn-outline-secondary btn-flag-spam-<%= node.id %>' href='mailto:moderators@publiclab.org?subject=Reporting+spam+on+Public+Lab&body=Hi,+I+found+this+item+that+looks+like+spam+or+needs+to+be+moderated:+<%= node.title.gsub(/ /,'+') %>+https://publiclab.org/n/<%= node.id %>+by+https://publiclab.org/profile/<%= node.author.username %>+Thanks!'>
+          <i class='fa fa-flag'></i>
+        </a>
+        <li data-toggle='tooltip' data-placement='top' title='Follow by tag or author' class='btn btn-outline-secondary btn-sm requireLogin nestedPopover' data-html='true' rel='popover' data-placement='left' data-content=&quot;    <%= "No tags" if tagnames.nil? || tagnames.length == 0 %>    <% if tagnames %><% tagnames.each do |tagname| %><p style='margin-bottom:3px; overflow: hidden; text-overflow: ellipsis;'><a href='/subscribe/tag/<%= tagname %>' class='btn btn-outline-secondary btn-sm'><%= tagname %></a></p><% end %><% end %><hr /><i class='fa fa-user'></i><% if current_user && !current_user.following?(node.author) %>&nbsp;<a class='btn btn-sm' href='/relationships?followed_id=<%= node.author.id %>' data-method='post' > <%= node.author.name %></a><% else %>  &nbsp;<%= node.author.name %><% end %>&quot;>
+          <i class='fa fa-user-plus' aria-hidden='true'></i>
+        </li>
+    
+         <a href='/notes/delete/<%= node.id %>' class='btn btn-outline-secondary btn-sm' data-confirm='Are you sure?'>
+          <i class='fa fa-trash'></i><span class='d-none d-lg-inline'> Delete</span>
+        </a>
+        <a href='/notes/raw/<%= node.id %>' class='btn btn-outline-secondary btn-sm'>
+          <i class='fa fa-code'></i><span class='d-none d-lg-inline'> Raw</span>
+        </a>
+        <% if logged_in_as(['admin', 'moderator']) %>
+          <% if node.type == 'note' %>
+            <a class='btn btn-outline-secondary btn-sm' href='/moderate/spam/<%= node.id %>'><i class='fa fa-ban-circle'></i> Spam</a>
+          <% elsif node.type == 'page' %>
+            <a class='btn btn-outline-secondary btn-sm' href='/moderate/revision/spam/<%= @revision.vid %>'><i class='fa fa-ban-circle'></i> Spam revision</a>
+          <% end %>
+        <% end %>
+      </div>
+    <% end %>
+    
+    <br />
+    <hr style='clear:both;' />
+    
+    <div>
+      <p><a id='print-command-no-links'><i class='fa fa-print'></i> Print without page links</a></p>
+      <p><a id='print-command-3-col'><i class='fa fa-print'></i> Print in 3-column layout</a></p>
+    </div>
+    ">
+      <span class="ff fa fa-copy"></span>
```

Comment:
This is the only one I'm not sure of. Maybe this should be a `...` menu instead? I'm just worried the 2-pages icon isn't clear enough. What do you think?

Reply:
So What does that copy icon in style guide mean ? I thought copy icon looks like pages , so its something similar to print of 2 pages and other stuff menu.

Reply:
Or should I just remove copy icon from this and just have `. . .` menu ?

Reply:
Hi @jywarren The style guide doesn't seem to have this menu icon in sidebar ? Instead it's usually in main left column.

![Screenshot from 2019-07-25 15-15-57](https://user-images.githubusercontent.com/26685258/61864591-4e903080-aeef-11e9-9f48-bbdb41fab52b.png)